app/views/conversations/show.html.erb

```diff
@@ -1,15 +1,100 @@
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind", selected: true %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+
+        <div class="text-white text-opacity-40 text-xs mx-2 mt-10 mb-2">
+          Yesterday
+        </div>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+      </div>
+
+      <div id="left-footer" class="flex-none flex text-white mb-4 mt-3">
+        <div class="h-10 flex pr-2 rounded-lg hover:bg-zinc-400 hover:bg-opacity-25 justify-between items-center w-full group cursor-pointer mb-1">
+          <div id="profile" class="ml-2 mr-2 w-7 h-7 bg-white rounded-full"></div>
+          <div class="flex-1 text-white truncate">
+            Keith Schacht
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+  <div id="right-column" class="flex flex-1 flex-col h-screen bg-white dark:bg-graphite">
+    <!-- Header that does not grow -->
+    <div id="wide-header" class="flex-none justify-between ml-2 mt-2 mb-3 h-10 text-black dark:text-white text-lg hidden md:flex">
+      <%= render 'gpt_header', conversation: @conversation %>
+
+      <div id="right-aligned-content" class="hover:bg-gray-100 cursor-pointer inline-flex mr-5 border border-gray-300 rounded-lg px-2 py-2 mt-1 tooltip tooltip-bottom hover:tooltip-open" data-tip="Share">
+        <%= icon "arrow-up-tray", variant: :outline, size: 18, class: 'text-black' %>
+      </div>
+    </div>
+
+    <div id="narrow-header" class="flex-none h-14 text-lg text-center bg-white dark:bg-zinc-800 md:hidden">
+      <div class="flex-1 mb-2 border-b border-zinc-300 dark:border-zinc-600 text-black dark:text-white dark:bg-graphite flex items-center justify-center">
+        <%= render 'gpt_header', conversation: @conversation %>
+      </div>
+    </div>
+
+    <div id="right-content-container" class="flex flex-col flex-grow overflow-auto">
+      <div id="right-content" class="flex-grow overflow-auto">
+        <div class="w-full md:max-w-none lg:max-w-[700px] xl:max-w-[810px] mx-auto mt-2">
```

Comment:
In general specifying widths like these makes me a little suspicious. Tailwind has a weird falling-over point where they don't specify sizes above a certain threshold, but I think they do it on purpose.

Regardless, does it make sense to add any of the sizes you've hard coded here into our tailwind config so it'll just generate classes for us?

Reply:
Response design isn’t my super strength. Is there a way around it explicitly specifying widths? The CSS issue I’m tying to solve is that ChatGPT varies the column size at different breakpoints. It looks pretty nice, and I’m in the mode of “perfectly copy, don’t try to improve (yet)” so I am just blindly making everything line up perfectly. So I think I need to explicitly specify the width.

I didn’t realize you could put sizes in the tailwind config, or maybe I did see that in the docs but I haven’t done it yet for anything. Since this was the only place I was using these, it didn’t feel worth creating a custom utility class for it.

Reply:
Every part of tailwind I know of [has the ability to customize your theme](https://tailwindcss.com/docs/width#customizing-your-theme). There's a lot of design value to sticking to a strict theme because it creates a lot of consistency with the design. Tailwind overrides then stick out as a place that something is going on that could look different than the rest of the site.

Totally fine rolling forward.